### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by mouuii**
refactor some code

### CODE REVIEW

**Review by kilosonc — APPROVED** [review]

File: pkg/application/service/service.go [hunk 1]
```diff
@@ -3,11 +3,11 @@ package service
...
 
-	applicationmanager "github.com/horizoncd/horizon/pkg/application/manager"
+	"github.com/horizoncd/horizon/pkg/application/manager"
 	"github.com/horizoncd/horizon/pkg/application/models"
-	groupservice "github.com/horizoncd/horizon/pkg/group/service"
-	"github.com/horizoncd/horizon/pkg/param/managerparam"
+	groupsvc "github.com/horizoncd/horizon/pkg/group/service"
 )
 
 type Service interface {
```
Comment (fdiskbrain) [inline pkg/application/service/service.go:13]: Can rename `Servicer`

**Review by cubxxw — APPROVED** [review]
LGTM

File: pkg/application/service/service.go (hunk 1 above)
Comment (mouuii) [reply pkg/application/service/service.go:13]: there already service struct and interface in this pkg

**Review by fdiskbrain — APPROVED** [review]
LGTM

**Comment by xuzhu-591** [comment]
LGTM

**Review by xuzhu-591 — APPROVED** [review]

**Pull request merged**